content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+    caption="CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod.png"
+    caption="Memory usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="resource-usage.png"
+    caption="CPU and Memory usage for cluster"
+    >}}
+
+Let’s start by looking at CPU usage by pod. In the sidecar scenarios, the containers utilizing the most CPU resources are the [frontend](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/frontend) service sidecars, which is main server hit by the load generators and also responsible for making any subsequent calls to other underlying microservices.
+
+In the ambient scenarios, a particular ztunnel's performance depends on the distribution of services across the nodes in the cluster. In the ambient with both ztunnel and waypoint proxy scenario, there is an increase in average CPU usage per ztunnel as traffic is now directed to waypoint proxies. The waypoint proxies themselves consume more resources than their sidecar counterparts since there are only three instances processing all traffic in the namespace versus one per pod in the sidecar scenario.
+
+Next is memory usage by pod. In all Istio scenarios, memory usage stays relatively constant for each pod during the test runs. In both ambient scenarios, the L4 ztunnel consumes less memory than a sidecar. In fact, in the L4 only scenario, almost all three ztunnel instances consume less memory than a single sidecar. Waypoint proxies consume a similar amount of resources as sidecars do, but once again see a minor increase due to the same reasons outlined above for CPU. So what do these per pod usages mean for the totals across the cluster? Everything.
+
+{{< image width="100%"
+    link="total-cpu.png"
+    caption="Total CPU usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="total-memory.png"
+    caption="Total memory usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="cpu-by-pod-stacked.png"
+    caption="Stacked CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod-stacked.png"
+    caption="Stacked memory usage by pod for sidecar and ambient pods"
+    >}}
+
+Looking at total CPU and memory utilization, we have to remember that in sidecar scenarios there are 144 sidecar containers required (48 services, 3 replicas each), while in ambient, only three ztunnel containers and 12 waypoint proxies are required. The stacked CPU and memory by pod graphs are excellent at highlighting just how many additional containers are present between scenarios. Memory usage of the Istio dataplane in the ztunnel-only ambient scenario uses *less than 1%* of what is used in sidecar scenarios, and still only 12% when waypoints are added. Looking at CPU, ztunnel uses *18%* of what sidecar scenarios require, and 60% when waypoints are deployed.
```

Comment:
the numbers here don't match what is in the table, pls clarify.

* Looking at CPU, ztunnel uses *18%* of what sidecar scenarios require, and 60% when waypoints are deployed * 

In your table, it showed 98% saving of cpu for L4 ($1 vs $61) and almost 90% saving for L4 + L7 ($6 vs $61).